src/test/java/com/google/cloud/spanner/pgadapter/ITQueryTest.java

```diff
@@ -189,7 +189,7 @@ public void basicSelectTest() throws Exception {
     // Check exact message results.
     byte[] rowDescriptionData = {
       0, 4, 105, 100, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 20, 0, 8, 0, 0, 0, 0, 0, 0, 110, 97, 109, 101,
-      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, -1, -1, 0, 0, 0, 0, 0, 0, 97, 103, 101, 0, 0, 0, 0, 0, 0, 0,
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 4, 19, -1, -1, 0, 0, 0, 0, 0, 0, 97, 103, 101, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
Curious on why did this change

Reply:
It was wrong. This is basically the `ResultSetMetaData` of a PG ResultSet, and there was a type mapping missing, which meant that the column was included in the metadata as `Oid.UNSPECIFIED` (zero), while it actually was a `Oid.VARCHAR` (1043).